RankUpper-56/src/main/java/br/net/fabiozumbi12/RankUpper/config/PlayerStatsDB.java

```diff
@@ -125,17 +133,17 @@ public void savePlayersStats(){
                 stmt.setString(7, stat.getValue().LastVisit);
                 stmt.setString(8, stat.getValue().PlayerName);
                 stmt.setInt(9, stat.getValue().TimePlayed);
-
-                stmt.executeUpdate();
+                stmt.addBatch();
```

Comment:
Batch the preparedstatements instead of running them once for every online player